packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.tsx

```diff
@@ -18,7 +18,15 @@ import { Responses } from './Responses';
 export type HttpOperationProps = DocsComponentProps<IHttpOperation>;
 
 const HttpOperationComponent = React.memo<HttpOperationProps>(
-  ({ className, data: unresolvedData, headless, uri, hideTryIt, hideTryItPanel, allowRouting = false }) => {
+  ({
+    className,
+    data: unresolvedData,
+    uri,
+    hideTryIt,
+    hideTryItPanel,
+    allowRouting = false,
+    customStyle = { noHeading: false },
```

Comment:
Unfortunately you can't default options like this. The problem is that anyone who passes even one prop will override this. I mean if `customStyle={{asd: true}}` then you lose `noHeading: false`.

The solution is usually to use [_.defaults](https://lodash.com/docs/4.17.15#defaults) inside the function.

Reply:
Also you don't need that default as your prop is negative. Undefined acts pretty much the same, it's also falsey.

Reply:
Also, I will change `hidePoweredBy` to `showPoweredBy`- first of all hiding is the default behavior, and secondly it allows to get rid of setting these default values